samples/server.c

```diff
@@ -263,16 +263,17 @@ migration_device_state_transition(vfu_ctx_t *vfu_ctx, vfu_migr_state_t state)
             server_data->migration.pending_bytes = server_data->bar1_size + sizeof(time_t); /* FIXME BAR0 region size */
             break;
         case VFU_MIGR_STATE_PRE_COPY:
-            /* TODO must be less than size of data region in migration region */
             server_data->migration.pending_bytes = server_data->bar1_size;
             break;
         case VFU_MIGR_STATE_STOP:
             /* FIXME should gracefully fail */
             assert(server_data->migration.pending_bytes == 0);
             break;
         case VFU_MIGR_STATE_RESUME:
+            server_data->migration.pending_bytes = server_data->bar1_size + sizeof(time_t);
```

Comment:
should discuss why/if we have pending_bytes at all still

Reply:
Actually maybe `bytes_read` is not a good name given that we use it to track how much we've written too (not at the same time of course), maybe something like `bytes_transferred`? What do you think?